packages/core/hooks/useOpenWithMenuItems/index.tsx

```diff
@@ -9,6 +9,90 @@ import { interaction, metadata } from "../../state";
 
 import styles from "./useOpenWithMenuItems.module.css";
 
+interface WebApps {
+    neuroglancer: IContextualMenuItem;
+    simularium: IContextualMenuItem;
+    volumeviewer: IContextualMenuItem;
+    volview: IContextualMenuItem;
+}
+
+const WEB_APPS = (fileDetails?: FileDetail): WebApps => ({
+    volumeviewer: {
+        key: "3d-web-viewer",
+        text: "3D Web Viewer",
+        title: `Open files with 3D Web Viewer`,
+        href: `https://volumeviewer.allencell.org/viewer?url=${fileDetails?.cloudPath}/`,
+        disabled: !fileDetails?.path,
+        target: "_blank",
+    },
+    neuroglancer: {
+        key: "neuroglancer",
+        text: "Neuroglancer",
+        title: `Open files with Neuroglancer`,
+        href: `https://neuroglancer-demo.appspot.com/#!{%22layers%22:[{%22source%22:%22zarr://${fileDetails?.cloudPath}%22,%22name%22:%22${fileDetails?.name}%22}]}`,
+        disabled: !fileDetails?.path,
+        target: "_blank",
+    },
+    simularium: {
+        key: "simularium",
+        text: "Simularium",
+        title: `Open files with Simularium`,
+        href: `https://simularium.allencell.org/viewer?trajUrl=${fileDetails?.cloudPath}`,
```

Comment:
should this one also be disabled on `!fileDetails?path` or not an issue for this one?

Reply:
Ah nope it should I missed that

Reply:
Done in https://github.com/AllenInstitute/biofile-finder/pull/281/commits/6fc3624ebf1c7f9ccff8ebbcef6c0fab6100681b